### PR TITLE
Grafana: add a runtime/Erlang/BEAM dashboard

### PR DESCRIPTION
**Pull request by lukebakken**
Also update the other dashboards by opening in Grafana v11.2.2 and ensuring they work as expected.

### CODE REVIEW

File: deps/rabbitmq_prometheus/docker/grafana/dashboards/Erlang-Distribution.json [hunk 1]
```diff
@@ -388,7 +408,7 @@
       "collapsed": false,
       "datasource": {
         "type": "prometheus",
-        "uid": "${DS_PROMETHEUS}"
+        "uid": "adz3cj19a7apsf"
```
Comment (lukebakken) [inline deps/rabbitmq_prometheus/docker/grafana/dashboards/Erlang-Distribution.json:411]: My guess is I should revert these changes...

**Review by mkuratczyk — APPROVED** [review]
Cool, thanks. I fixed one more panel. I'm not super happy about it, but can't think of a better visualization. At least it shows meaningful data now.

**Comment by lukebakken** [comment]
@mkuratczyk thanks for the review!
